pkg/cache/cache_test.go

```diff
@@ -181,6 +181,7 @@ func TestStoreRelocationMapping(t *testing.T) {
 			assert.Equal(t, tc.wantedReloPath, cb.RelocationFilePath, "didn't get expected path for store")
 
 			cb, _, err = c.FindBundle(tc.tag)
+			assert.NoError(t, err, fmt.Sprintf("didn't expect find bundle error for test %s", tc.tag))
```

Comment:
You don't have to change this but I wanted you to be aware that the assert.NoError function will handle formatting for you. So this line can be simplified to 
```suggestion
			assert.NoError(t, err, "didn't expect find bundle error for test %s", tc.tag)
```